tools/run-dev.py

```diff
@@ -345,6 +346,14 @@ def print_listeners() -> None:
     proxy_warning = f"Only the proxy port ({proxy_port}) is exposed."
     print(WARNING + "Note to Vagrant users: " + ENDC + proxy_warning + '\n')
 
+if options.test:
+    do_one_time_webpack_compile()
+else:
+    start_webpack_watcher()
+
+for cmd in server_processes():
+    subprocess.Popen(cmd)
```

Comment:
I'm not that excited about the model change to have `webpack` not be started in this loop, in part because I expect we'll end up adding more features to this loop (like auto-restarting crashed processes).  But I suppose we could address that later.

Reply:
Yeah, I'm not too worried about it either way.  Regardless of what we do here, the webpack code is gonna be very tool-specific.